floof.yaml

```diff
@@ -0,0 +1,59 @@
+default:
+  - command:
+      run: cargo run --bin export_schema -- ../frontend/build/schema.graphql
+      workdir: backend
+  - concurrently:
+    - http:
+        proxy: localhost:3080
```

Comment:
I get the following error in this line:

```
Error: failed to load configuration from 'floof.yaml'

Caused by:
    0: failed to deserialize YAML file
    1: default[1].concurrently[0].http: expected one address, but found 2 at line 7 column 14
```

Reply:
Hu 0.o you sure you have installed `floof` from the newest `master`?

Reply:
Interesting! I wasn't aware of that behavior. It should be fixed now.

Reply:
Now I get this: 
![image](https://user-images.githubusercontent.com/123272/92085057-8e029f00-edc8-11ea-9255-ae7414b53b74.png)

Seems to be related to the above, as it tries to access the IPv6 address? :thinking:

Reply:
Interesting. I now changed `localhost` to `127.0.0.1` as the former might simply be underspecified. Tobira by default listens on 127.0.0.1, so this should be fine I guess.